.buildbot.sh

```diff
@@ -16,3 +24,9 @@ ulimit -d $((1024 * 1024 * 8)) # 8 GiB
 
 /usr/bin/time -v python3 x.py test --stage 2 --config .buildbot.config.toml --exclude rustdoc-json --exclude debuginfo
 
+# Build and test yksom
+git clone --recursive https://github.com/softdevteam/yksom
+cd yksom
+rustup toolchain link alloy build/x86_64-unknown-linux-gnu/stage1
```

Comment:
Should this line go before `cd yksom`?

Reply:
Oops. Yes. Fixed here: 4d03af851ba41a3e91ece9a389b95cd9e0ba26c1